Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fafnir"
-version = "0.2.0"
+version = "0.2.1"
```

Comment:
This crate version has not been upgraded with the latest releases (see https://github.com/QwantResearch/fafnir/releases/tag/v0.3.4). :-/
So we should jump to `0.3.5` to be consistent with the versions used as tags on docker images.

Reply:
Not sure to understand why it'd be interesting to do so but ok.

Reply:
We could just ignore it, as Fafnir is only used as a binary anyway. But now that this PR is open, let's clean it up :smiley: